projects/storefrontlib/src/cms-components/checkout/components/payment-method/payment-form/payment-form.component.html

```diff
@@ -1,202 +1,394 @@
               type="text"
               class="form-control"
-              id="cVVNumber"
               required
-              formControlName="cvn"
+              formControlName="cardNumber"
             />
+            <cx-form-errors
+              [control]="paymentForm.get('cardNumber')"
+            ></cx-form-errors>
           </label>
         </div>
-      </div>
 
-      <div class="form-group" *ngIf="setAsDefaultField">
-        <div class="form-check">
-          <label>
-            <input
-              type="checkbox"
-              class="form-check-input"
-              (change)="toggleDefaultPaymentMethod()"
-            />
-            <span class="form-check-label">{{
-              'paymentForm.setAsDefault' | cxTranslate
-            }}</span>
-          </label>
+        <div class="row">
+          <div class="form-group col-md-8">
+            <label>
+              <span class="label-content">{{
+                'paymentForm.expirationDate' | cxTranslate
+              }}</span>
+              <div class="cx-payment-form-exp-date">
+                <div class="cx-payment-form-exp-date-wrapper">
+                  <ng-select
+                    [searchable]="false"
+                    [clearable]="false"
+                    [items]="months"
+                    placeholder="{{ 'paymentForm.monthMask' | cxTranslate }}"
+                    (change)="monthSelected($event)"
+                    formControlName="expiryMonth"
+                  >
+                  </ng-select>
+                  <cx-form-errors
+                    [control]="paymentForm.get('expiryMonth')"
+                  ></cx-form-errors>
+                </div>
+                <div class="cx-payment-form-exp-date-wrapper">
+                  <ng-select
+                    [searchable]="false"
+                    [clearable]="false"
+                    [items]="years"
+                    placeholder="{{ 'paymentForm.yearMask' | cxTranslate }}"
+                    (change)="yearSelected($event)"
+                    formControlName="expiryYear"
+                  >
+                  </ng-select>
+                  <cx-form-errors
+                    [control]="paymentForm.get('expiryYear')"
+                  ></cx-form-errors>
+                </div>
+              </div>
+            </label>
+          </div>
+          <div class="form-group col-md-4">
+            <label>
+              <span class="label-content">
+                {{ 'paymentForm.securityCode' | cxTranslate }}
+                <cx-icon
+                  [type]="iconTypes.INFO"
+                  class="cx-payment-form-tooltip"
+                  placement="right"
+                  title="{{ 'paymentForm.securityCodeTitle' | cxTranslate }}"
+                  alt=""
+                ></cx-icon>
+              </span>
+              <input
+                type="text"
+                class="form-control"
+                id="cVVNumber"
+                required
```

Comment:
I haven't seen this param on other forms. Why we have it here?

Reply:
This attribute was also present in 2 other forms. Removed